README.md

```diff
@@ -1,20 +1,26 @@
-# echidna
+# Echidna: A Fast Smart Contract Fuzzer <a href="https://raw.githubusercontent.com/crytic/echidna/master/echidna.png"><img src="https://raw.githubusercontent.com/crytic/echidna/master/echidna.png" width="75"/></a>
 
 ![Build Status](https://github.com/crytic/echidna/workflows/CI/badge.svg)
-![echidna logo](echidna.png)
 
 Echidna is a weird creature that eats bugs and is highly electrosensitive (with apologies to Jacob Stanley)
 
-More seriously, Echidna is a Haskell library designed for fuzzing/property-based testing of EVM code. It supports relatively sophisticated grammar-based fuzzing campaigns to falsify a variety of predicates.
+More seriously, Echidna is a Haskell program designed for fuzzing/property-based testing of Ethereum smarts contracts. It uses a relatively sophisticated grammar-based fuzzing campaigns based on a contract ABI to falsify Solidity assertions or user-defined predicates. We designed Echidna with modularity in mind, so it can be easily extended to include new mutations or test specific contracts in specific cases.
 
 ## Features
 
 * Generates inputs tailored to your actual code
-* Optional coverage guidance to find deeper bugs
+* Optional corpus collection, mutation and coverage guidance to find deeper bugs
+* Curses-based retro UI, text only or JSON output
 * Automatic testcase minimization for quick triage
 * Seamless integration into the development workflow
-* Fast
-* Beautiful logo
+* Automatic detection of transactions with high-gas consumption
+* Extended testing of complex contracts using Etheno and Truffle
+
+.. and [a beautiful high-resolution handcrafted logo](https://raw.githubusercontent.com/crytic/echidna/master/echidna.png).
+
+## Screenshot
+
+<a href="https://trailofbits.files.wordpress.com/2020/03/image5.png"><img src="https://trailofbits.files.wordpress.com/2020/03/image5.png" width="800"/></a>
```

Comment:
Maybe move the screenshot to the repository?

Reply:
We could, but I prefer to link blog posts to link their screenshots. So we are forced to write a blog post about our tool from time time.